### PR TITLE
Adding a sample on how to use the custom serializer.

### LINKED ISSUE #559: Add sample using custom serializer
Add a sample project using a custom serializer for new users.

### PR DESCRIPTION
**Pull request by j82w**
# Pull Request Template

## Description

Adding a sample on how to use the custom serializer.

## Closing issues

Put closes #559

### CODE REVIEW

**Comment by j82w** [comment]
@nheinbaugh here is the sample for the custom serializer

File: Microsoft.Azure.Cosmos.Samples/CodeSamples/CustomSerializer/JsonSerializerIgnoreNull.cs [hunk 1]
```diff
@@ -0,0 +1,58 @@
+﻿namespace Cosmos.Samples.Shared
+{
+    using System;
+    using System.IO;
+    using System.Text;
+    using Microsoft.Azure.Cosmos;
+    using Newtonsoft.Json;
+
+    class JsonSerializerIgnoreNull : CosmosSerializer
```
Comment (kirankumarkolli) [inline Microsoft.Azure.Cosmos.Samples/CodeSamples/CustomSerializer/JsonSerializerIgnoreNull.cs]: One more option is to make it generic to take JsonSerializerSettings and we can use it as reference for V2 -> V3 migration. Thoughts?

Comment (j82w) [reply Microsoft.Azure.Cosmos.Samples/CodeSamples/CustomSerializer/JsonSerializerIgnoreNull.cs]: I created a separate issue to track that. https://github.com/Azure/azure-cosmos-dotnet-v3/issues/551

Comment (kirankumarkolli) [reply Microsoft.Azure.Cosmos.Samples/CodeSamples/CustomSerializer/JsonSerializerIgnoreNull.cs]: JSON.net implementation as public contract will force its dependency even in future when implementation moved to .NET core JSON parser.

**Comment by kirankumarkolli** [comment]
#551 already covers it and its an adoption blocker needs attention.
Post #551 closure this samples has very little/none left.

**Pull request closed without merging**